pkg/controller/stacks/stack/stack.go

```diff
@@ -488,6 +492,81 @@ func (h *stackHandler) createPersonaClusterRolesCRDHandler() crdHandler {
 
 }
 
+func stringSlicesEqual(a, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+
+	for i := range a {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+// policiesEqual verifies that two RBAC policy rules match
+//
+// TODO(displague) Any difference in the order of the policies array fields
```

Comment:
That makes sense. I could permit subset matches too if I change this from `policyEqual` to `policyIsSubset`. 

This would allow a Stack resource to state, 'I don't need secrets but do need configmaps and events'.

Any thoughts on taking that direction now vs later?

Reply:
I think we can proceed with what you have implemented and see if we need to expand out support/functionality in response to real use cases.